### PR TITLE
Build Okio for WASM

### PR DESCRIPTION
**Pull request by swankjesse**

### CODE REVIEW

File: okio/build.gradle.kts [hunk 1]
```diff
@@ -166,26 +160,9 @@ kotlin {
   }
 }
 
-tasks {
```
Comment (swankjesse) [inline okio/build.gradle.kts:169]: * [ ] Restore the BND plugin. It was colliding with WASM for some reason?

File: kotlin-js-store/yarn.lock [hunk 2]
```diff
@@ -12,6 +12,51 @@
   resolved "https://registry.yarnpkg.com/@discoveryjs/json-ext/-/json-ext-0.5.7.tgz#1d572bfbbe14b7704e0ba0f39b74815b84870d70"
   integrity sha512-dBVuXR082gk3jsFp7Rd/JI4kytwGHecnCoTtXFb7DB6CNHp4rg5k1bhg0nWdLGLnOV71lmDzGQaLMy8iPLY0pw==
 
+"@jridgewell/gen-mapping@^0.3.0":
```
Comment (oldergod) [inline kotlin-js-store/yarn.lock:15]: How does one update this file?

Comment (swankjesse) [reply kotlin-js-store/yarn.lock:15]:
```
./gradlew kotlinUpgradeYarnLock
```

**Comment by swankjesse** [comment]
This doesn’t yet support a FileSystem!

**Review by JakeWharton — APPROVED** [review]

**Pull request merged**